ci/meta.yml

```diff
@@ -54,11 +54,13 @@ meta:
             cp ../../../terraform-config/terraform.tfvars .
             terraform init
             terraform workspace show
+            export TF_VAR_packer_egress_test_zip="{base_path = \"../packer-egress-test-release\", version = \"$(cat ../packer-egress-test-release/version)\"}"
             terraform plan -out terraform.plan
             terraform apply -auto-approve terraform.plan
         inputs:
         - name: aws-concourse
         - name: terraform-config
+        - name: packer-egress-test-release
```

Comment:
this appears to be missing from the `get` step in each job

Reply:
Yeah, the PR check caught that. Fixed.